### PR TITLE
BS-53-barchain-barhopping-implementation

### PR DESCRIPTION
**Pull request by SkyMazter**
First instance of manual barchaining, 
  - added functionality to the "Start Hopping" button when opening a more info tab under quick info
  - Refactored site to allow for a modular side panel
  - added mapslice and barHopSlice
  - removed side panel appearing when a marker is tapped
  - map switches between chaining and standard mode
  - fixed visual bug with the recenter button

### CODE REVIEW

File: frontend/src/types/fetchCall.tsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
-const baseURL = "https://barspot-server.onrender.com";
+// const baseURL = "https://barspot-server.onrender.com";
+const baseURL = "http://localhost:3001";
```
Comment (DFLotus) [inline frontend/src/types/fetchCall.tsx:2]: Revert back to prod

File: frontend/src/pages/Profile.tsx [hunk 2]
```diff
@@ -202,10 +202,13 @@ const Profile = () => {
 
 
     return (
-        <div className="profile-page">
+        <div className="profile-page" style={{
+            backgroundColor: "#646463",
+            height: "100vh"
```
Comment (DFLotus) [inline frontend/src/pages/Profile.tsx:207]: Why have the height css?

File: frontend/src/pages/MapView.tsx [hunk 3]
```diff
@@ -67,7 +106,12 @@ const MapView = () => {
     setOffCanvas(false);
     resetMap();
   };
-
+  // const handleOnMouseEnter = () => {
```
Comment (DFLotus) [inline frontend/src/pages/MapView.tsx:109]: Remove comments

File: frontend/src/pages/MapView.tsx [hunk 4]
```diff
@@ -132,24 +188,37 @@ const MapView = () => {
...
-    const directionsService = new google.maps.DirectionsService();
-    const results = await directionsService.route({
-      origin: startingPoint,
-      destination: endingPoint,
-      travelMode: google.maps.TravelMode.WALKING,
-    });
-
-    setDirections(results);
+    if (isChaining) {
+      console.log(chainedDirections);
```
Comment (DFLotus) [inline frontend/src/pages/MapView.tsx:197]: Remove console.log

File: frontend/src/main.tsx [hunk 5]
```diff
@@ -9,33 +9,47 @@ import { createBrowserRouter, RouterProvider } from "react-router-dom";
...
+    {
+      path: "/login",
+      element: <Login />,
+    },
+    {
+      path: "/sign-up",
+      element: <CreateNewAcc />,
+    },
+  ]
+  // { basename: '/bar-spot/' }
```
Comment (DFLotus) [inline frontend/src/main.tsx:51]: Revert change for prod

File: frontend/src/components/QuickInfo.tsx [hunk 6]
```diff
@@ -16,13 +16,16 @@ import {
...
 import { barMenuInfo } from "../types/types";
 import MoreInfo from "./MoreInfo";
+import { useData } from "../pages/MapView";
 
-interface Props {
-  barData: barMenuInfo[];
-  handleRecenter: () => void;
-}
+// interface OutletContextType {
+//   yelpData: barMenuInfo[];
```
Comment (DFLotus) [inline frontend/src/components/QuickInfo.tsx:22]: Remove comments

File: frontend/src/pages/Profile.tsx (hunk 2 above)
Comment (SkyMazter) [reply frontend/src/pages/Profile.tsx:207]: Without it there is a white rectangle at the bottom that ruins the visuals

Comment (DFLotus) [reply frontend/src/pages/Profile.tsx:207]: Understood

**Review by DFLotus — APPROVED** [review]
Amazing Job Oscar!

**Pull request merged**